src/main/java/seedu/planner/model/util/ModuleUtil.java

```diff
@@ -1,26 +1,21 @@
 package seedu.planner.model.util;
 
-//TODO
-
 /**
  * Helper functions for handling module.
  */
 public class ModuleUtil {
+    public static final String MODULE_CODE_REGEX = "^[A-Z]{2,3}[0-9]{4}[A-Z]{0,1}$";
```

Comment:
```suggestion
    public static final String MODULE_CODE_REGEX = "^[A-Z]{2,3}\\d{4}[A-Z]{0,2}$";
```

Some module codes can have 2 alphabets as suffix. `\d` means `[0-9]`.